pkg/controllers/nodeclaim/termination/metrics.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package termination
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	crmetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
+
+	"sigs.k8s.io/karpenter/pkg/metrics"
+)
+
+func init() {
+	crmetrics.Registry.MustRegister(NodeClaimTerminationDuration)
+}
+
+var NodeClaimTerminationDuration = prometheus.NewHistogramVec(
```

Comment:
I still think this needs to be something like "instance_termination_duration" to decouple the concept of the NodeClaim CR and the CloudProvider instance itself